windows/src/main/csharp/ch/cyberduck/ui/winforms/PreferencesForm.cs

```diff
@@ -684,6 +684,30 @@ public int ConnectionTimeout
             get { return Convert.ToInt32(connectionTimeoutUpDown.Value); }
             set { connectionTimeoutUpDown.Value = value; }
         }
+        public int ConnectionTimeoutMin
+        {
+            set
+            {
+                connectionTimeoutUpDown.Minimum = new decimal([
+                    value,
+                    0,
+                    0,
+                    0
+                ]);
+            }
```

Comment:
``` suggestion
            set => connectionTimeoutUpDown.Minimum = value;
```
No need to create either allocate an array, or create an inline Span.
The original designer generated code was subpar at best, directly assigning a value is better and less error-prone.